openshift-knative-operator/cmd/operator/kodata/ingress/1.10/kourier/1-net-kourier.yaml

```diff
@@ -401,7 +401,7 @@ spec:
               cpu: 200m
               memory: 200Mi
             limits:
-              cpu: 500m
+              cpu: "1"
               memory: 500Mi
       restartPolicy: Always
       serviceAccountName: net-kourier
```

Comment:
@ReToCode @skonto @nak3 Hi guys, can you pls confirm this's desired change on `release-1.31` branch as well. It's pulled through `make generated-files`.

Reply:
👍  this change is fine.